context/src/junit/validator.rs

```diff
@@ -86,14 +86,15 @@ pub fn validate(report: &Report) -> JunitReportValidation {
         if let Some(raw_test_suite_id) = test_suite.extra.get("id") {
             let test_case_id = uuid::Uuid::parse_str(raw_test_suite_id).unwrap_or_default();
             if test_case_id.get_version() != Some(uuid::Version::Sha1) {
-                test_suite_validation.add_issue(JunitValidationIssue::Invalid(
-                    JunitTestSuiteValidationIssueInvalid::TestSuiteInvalidId(
+                test_suite_validation.add_issue(JunitValidationIssue::SubOptimal(
+                    JunitTestSuiteValidationIssueSubOptimal::TestSuiteInvalidId(
```

Comment:
Ran this by Gabe - the current ETL validation logic ignores the `id` attribute on test suites, so calling non-uuid IDs invalid here would cause us to start throwing away some test suites (there are some junits have something like `<testsuite id="0" ... >`). De-escalating to suboptimal for now so we at least continue to process them, even though it looks like we're not using test suite ID anywhere downstream in the ETL